website/blog/2022/03/04/apisix-clickhouse-makes-logging-easier.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Making It Simple, Apache APISIX Integrates ClickHouse to Improve Logging Efficiency"
```

Comment:
How about using `api gateway` instead of `apisix` in url?

Reply:
updated both file names